tests/NgTests/Db2CatalogTests.cs

```diff
@@ -1058,6 +1247,8 @@ public async Task RunInternal_CallsCatalogStorageLoadStringExactlyOnce()
                 top: top,
                 verbose: true,
                 Constants.MaxPageSize,
+                Constants.NoStoreCacheControl,
+                Constants.NoStoreCacheControl,
```

Comment:
Consider changing this to add clarity. 

```suggestion
                itemCacheControl: Constants.NoStoreCacheControl,
                finishedPageCacheControl: Constants.NoStoreCacheControl,
```

Reply:
Fixed.